cosmwasm/cli-utils.js

```diff
@@ -201,9 +205,16 @@ const addProposalOptions = (program) => {
     program.addOption(new Option('--deposit <deposit>', 'the proposal deposit'));
 };
 
+const addOptionalProposalOptions = (program) => {
+    program.addOption(new Option('-t, --title <title>', 'title of proposal (required for proposals)'));
+    program.addOption(new Option('-d, --description <description>', 'description of proposal (required for proposals)'));
+    program.addOption(new Option('--deposit <deposit>', 'the proposal deposit'));
```

Comment:
this is a fixed amount. should just read from the config per env, and simplify UX?